seq2seq/trainer/supervised_trainer.py

```diff
@@ -290,6 +293,13 @@ def get_optim(optim_name):
     def get_batch_data(batch, **kwargs):
         input_variables, input_lengths = getattr(batch, seq2seq.src_field_name)
         target_variables = {'decoder_output': getattr(batch, seq2seq.tgt_field_name)}
+
+        # If available, also get provided attentive guidance data
+        if hasattr(batch, seq2seq.attn_field_name):
+            attention_variables = getattr(batch, seq2seq.attn_field_name)
+
+            target_variables['provided_attention'] = attention_variables
```

Comment:
In `get_batch_data` I add the provided attention to the `target_variables` variable, even though it's technically not a target variable. This was just the easiest way to make that the decoder receives this data somehow, without changing several function signatures.
Do you agree with this, or should I change it?

Reply:
Yes I think it's fine.